curriculum/challenges/english/blocks/workshop-shape-manager/69145aefce21f93ed77b626c.md

```diff
@@ -0,0 +1,366 @@
+---
+id: 69145aefce21f93ed77b626c
+title: Step 16
+challengeType: 25
+dashedName: step-16
+---
+
+# --description--
+
+You also need to toggle the result card on and off based on what is calculated and the result to show.
+
+For this, create a `toggleResultCard` function that takes a boolean type parameter called `show`. If `show` is true, add the `visible` class to `resultCard`. Otherwise, remove it.
+
+# --hints--
+
+You should create a `toggleResultCard` function.
+
+```js
+assert.isFunction(toggleResultCard)
+```
+
+Your `toggleResultCard` function should have a `show` parameter set to a type of `boolean`.
+
+```js
+assert.match(code, /\(show\s*:\s*boolean\)/)
+```
+
+Your `toggleResultCard` function should return `true` or `false` based on shape selection and deselection.
```

Comment:
This hint should be something like:
```suggestion
Your `toggleResultCard` function should add the `visible` class to `resultCard` when `show` is `true`, and remove it otherwise.
```